src/psyclone/transformations.py

```diff
@@ -1096,11 +1096,11 @@ def name(self):
         ''' Returns the name of this transformation as a string '''
         return "Move"
 
-    def _validate(self, node, location, position):
+    def _validate(self, node, location, position):  # pylint: disable=no-self-use
```

Comment:
I wouldn't worry about these pylint warnings since putting in the disable clause then makes the line too long for pep8!